store/storetest/channel_store.go

```diff
@@ -442,7 +442,7 @@ func testChannelStoreGetChannelsByIds(t *testing.T, ss store.Store) {
 	o1 := model.Channel{}
 	o1.TeamId = model.NewId()
 	o1.DisplayName = "Name"
-	o1.Name = "aa" + model.NewId() + "b"
+	o1.Name = "aa" + model.NewId()
```

Comment:
Are these remaining prefixes intentional?

Reply:
yes. remove the prefix tests will not pass